src/modules/27-platform/default-settings/interfaces/SettingType.types.ts

```diff
@@ -12,7 +12,8 @@ export type SettingCategory = SettingDTO['category']
 export enum SettingGroups {
   test_group_2 = 'test_group_2',
   PERSPECTIVES_PREFERENCES = 'perspectives_preferences',
-  TICKETING_PREFERENCES = 'ticketing_preferences'
+  TICKETING_PREFERENCES = 'ticketing_preferences',
+  ENABLED_NOTIFICATION_CHANNELS_GROUP_IDENTIFIER = 'enabled_notification_channels'
```

Comment:
NIT: Follow similar naming convention of have same key and value.
```suggestion
  ENABLED_NOTIFICATION_CHANNELS = 'enabled_notification_channels'
```